Xamarin.Forms.Platform.iOS/CollectionView/CollectionViewController.cs

```diff
@@ -42,7 +42,14 @@ public override UICollectionViewCell GetCell(UICollectionView collectionView, NS
 
 		public override nint GetItemsCount(UICollectionView collectionView, nint section)
 		{
-			return _itemsSource.Count;
+			var count = _itemsSource.Count;
+			if (_layout.IsDeterminingCellSize && count == 1)
```

Comment:
I would like a better idea for this I don't have one right now. 
So basically when you set EstimateItemSize on `DetermineCellSize` It will queue a call to GetItemsCount, since the source already has 1 item, but we called this method before adding a NativeCell to the UICollectionView, so to return the correct count we need to return 0.

Reply:
I may not be understanding this, but what happens if there are 2 items in the source? Why do we check specifically for `1` item? What if there is no items? Does this mean the size might be wrong? If the size is going to be incorrect, can we postpone the measuring until we have an item?